manual/C/ch_Finance-Quote.xml

```diff
@@ -240,6 +251,199 @@ SYNOPSIS
     </procedure>
   </sect1>
 
+  <sect1 id="finance-quote-helper">
+    <title>Helper Scripts for Update and Diagnostics</title>
+
+    <abstract>
+      <para>&app; provides several utilities that support you in your daily work with &app-fq;.
+        So there is a script with which you can check your symbols for the securities. Other
+        scripts perform the initial installation or keep &app-fq; up to date.
+        &app-fq; software module. These programs have names that begin with gnc-fq-*, and are
+        themselves written in the &app-perl; programming language. 
+      </para>
```

Comment:
I don't think "daily work" really applies here. The utilities aren't needed often and are mostly useful for troubleshooting.

I'd also leave off the bit about names beginning with `gnc-fq-*` since they're enumerated below.